README.md

```diff
@@ -94,30 +104,53 @@ $ grunt vendor
 $ grunt
 ```
 
+### 2. Run Elasticsearch
+
+To launch Elasticsearch, first find out where your Elasticsearch config file is located. You can do this with [Homebrew](http://brew.sh) using:
+
+```bash
+$ brew info elasticsearch
+```
+
+The last line of that output should be the command you need to launch Elasticsearch with the proper path to its configuration file. For example, it may look like:
+
+```
+$ elasticsearch --config=/Users/<YOUR MAC OSX USERNAME>/homebrew/opt/elasticsearch/config/elasticsearch.yml
+```
+
+### 3. Launch sheer to serve the site
+
 To work on the app you will need sheer running to compile the templates.
 There is also a `grunt watch` command that will recompile Less and JS
 on the fly while you're developing. To do this, run the following:
 
 ```bash
-# use the sheer virtualenv
+# Use the sheer virtualenv.
 $ workon sheer
 
-# index the latest content
+# Index the latest content from the API output of our WordPress back-end.
```

Comment:
Yes, I think so. Can you clarify how Django fits in? I'm forgetting. There's a Django-based API that content is being pulled from? Could this comment be:

```bash
# Index the latest content from the API output from a WordPress and Django back-end.
```

Also, is WordPress a hard requirement? Will the project run without it but just be devoid of content? Or it not run without it?